src/client/canvas/object/scene.ts

```diff
@@ -0,0 +1,42 @@
+import { computed } from 'mobx'
+import { Transform } from '../../math/transform'
+import { Group } from './group'
+import { GroupOpts } from './group'
+import { Object2d } from './object2d'
+import { action } from 'mobx'
+import { observable } from 'mobx'
+
+export type SceneOpts = GroupOpts
+
+export class Scene implements Object2d {
```

Comment:
I was thinking about this myself. The only things threejs adds to their scene object that isn't on an object3d is an overall background, fog and an override. I doubt we'll need any of those and they could just be applied to a parent shape appearance in our case. I do like referring to the root object as a scene type but that's literally it's only benefit. Thoughts on the latter?